### PR TITLE
feat:inspector support show text-like file

### PR DESCRIPTION
**Pull request by dogodo-cc**
Re: # https://github.com/cocos/3d-tasks/issues/15528

### Changelog

*inspector support show text-like file

-------

### Continuous Integration

This pull request:

* [ ] needs automatic test cases check.
  > Manual trigger with `@cocos-robot run test cases` afterward.
* [ ] does not change any runtime related code or build configuration
  > If any reviewer thinks the CI checks are needed, please uncheck this option, then close and reopen the issue.

-------

### Compatibility Check

This pull request:

* [ ] changes public API, and have ensured backward compatibility with [deprecated features](https://github.com/cocos/cocos-engine/blob/v3.5.0/docs/contribution/deprecated-features.md).
* [ ] affects platform compatibility, e.g. system version, browser version, platform sdk version, platform toolchain, language version, hardware compatibility etc.
* [ ] affects file structure of the build package or build configuration which requires user project upgrade.
* [ ] **introduces breaking changes**, please list all changes, affected features and the scope of violation.

### CODE REVIEW

File: editor/inspector/assets/common.js [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+.asset-text {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    /* it is necessary */
+    height: 0px;
+}
+.asset-text > ui-code {
+    flex: 1;
+}
```
Comment (doommm) [inline editor/inspector/assets/common.js]: 这里的样式和 html 的 `asset-unknown` 对不上？

File: editor/inspector/assets.js [hunk 2]
```diff
@@ -31,4 +31,5 @@ module.exports = {
     typescript: join(__dirname, './assets/typescript.js'),
     unknown: join(__dirname, './assets/unknown.js'),
     prefab: join(__dirname, './assets/prefab.js'),
+    "*": join(__dirname, './assets/common.js'),
```
Comment (arsen2010) [inline editor/inspector/assets.js]: 星号英文叫 asterisk，建为 asterisk.js 文件

File: editor/inspector/assets/common.js [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+const textLike = ['.atlas'];
+const { createReadStream } = require('fs');
+const ReadLine = require('readline');
+const { extname } = require('path');
+
+const MAX_LINES = 400;
+const MAX_LENGTH = 20000;
+
+exports.template = /* html */`
+<section class="asset-unknown">
```
Comment (arsen2010) [inline editor/inspector/assets/common.js]: 容器类名跟随文件名 asset-asterisk

File: editor/inspector/assets/common.js [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+    <ui-code language="xml"></ui-code>
+</section>`;
+
+exports.$ = {
+    container: '.asset-unknown',
+    code: 'ui-code',
+};
+
+exports.style = /* css */`
+.asset-text {
```
Comment (arsen2010) [inline editor/inspector/assets/common.js]: 需保持和 asset-asterisk 一致

File: editor/inspector/assets/common.js [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+
+    if (display === 'none') {
+        return;
+    }
+
+    const isTextLike = textLike.includes(extname(this.asset.name));
+    if (!isTextLike) { return; }
+
+
+    // Displays 400 lines or 20,000 characters
```
Comment (arsen2010) [inline editor/inspector/assets/common.js]: 以下逻辑抽取出来作为 textLikeRender(){ } ，后续易于扩展类似 textLike 的其他同级规则

Comment (arsen2010) [reply editor/inspector/assets/common.js]: renderContent 的逻辑是切换 ui-code 显示，可合并到 textLikeRender 里，通过是否动态生成 ui-code 元素并挂入容器实现

**Review by arsen2010 — APPROVED** [review]

**Review by VisualSJ — APPROVED** [review]

**Pull request merged**